technical/code_structuring.md

```diff
@@ -0,0 +1,57 @@
+# Code structuring
+
+## Standards for Java packages
+
+All packages in LocalMotion must be subpackages of `io.localmotion`.
```

Comment:
Do we have any specific reason for a package name to start with 'io'?. java naming convention for package name https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html

Reply:
we have the localmotion.io domain, so I propose to use io.localmotion... as package name